docs/guides/getting-started.md

```diff
@@ -5,39 +5,64 @@ description: |-
 ---
 
 # Getting started with Oasis Terraform Provider
+ArangoDB Oasis, the ArangoDB Cloud, provides ArangoDB databases as a Service (DBaaS). It enables you to use the entire functionality of an ArangoDB cluster deployment without the need to run or manage the system yourself.
 
+Terraform Provider Oasis is a plugin for Terraform that allows for the full lifecycle management of ArangoDB Cloud resources.
```

Comment:
We need naming. The line before it is called ArangoDB Oasis and here ArangoDB cloud